ni/heroku.py

```diff
@@ -18,6 +18,10 @@ def contrib_auth_token():
         return os.environ['GH_AUTH_TOKEN']
 
     @staticmethod
+    def contrib_secret_token():
+        return os.environ['GH_SECRET_TOKEN']
```

Comment:
normal log or log_exception? Because this PR assumes that we will always validate a request